app/assets/stylesheets/content.scss

```diff
@@ -1,8 +1,21 @@
 /************** CONTENT CONTAINER *******************************/
 
 #content {
+  background-color: #FFF;
```

Comment:
Color literals like `#FFF` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FFF` should be written as `#fff`